addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -954,19 +994,19 @@ Outputs:
     Condition: isAppStream
     Value: !Ref AppStreamStack
 
-  CFNEndpointSG:
-    Description: Security group of CFN endpoint
+  InterfaceEndpointSG:
+    Description: Security group of Interface endpoints
     Condition: isAppStream
-    Value: !Ref CfnEndpointSecurityGroup
+    Value: !Ref InterfaceEndpointSecurityGroup
     Export:
-      Name: !Join ['', [Ref: Namespace, '-CfnEndpointSecurityGroup']]
+      Name: !Join ['', [Ref: Namespace, '-InterfaceEndpointSG']]
 
-  VPCDefaultSG:
+  WorkspaceSG:
     Description: Default SG for VPC
```

Comment:
Workspace SG for AppStream

Reply:
Decided to keep it as is since we have quite a few resources in AppStream branch without AppStream in their name and we won't have non-AppStream and AppStream workspaces together